scripts/o3de/o3de/engine_template.py

```diff
@@ -2392,6 +2456,17 @@ def _run_create_gem(args: argparse) -> int:
                       args.module_id,
                       args.version)
 
+def _run_create_repo(args: argparse) -> int:
+    return create_repo(args.repo_path,
+                       args.repo_uri,
+                       args.repo_name,
+                       args.origin,
+                       args.origin_url,
+                       args.summary,
+                       args.additional_info,
+                       args.force,
+                       args.replace
```

Comment:
we should expose the no-register cli argument
```suggestion
                       args.replace,
                       args.no_register
```